### ISSUE TITLE
Lost all my hosts after adding one!

### ISSUE THREAD

**Issue by sylvoie** [blinksh/blink#1413]
Hello,

First, migration from Blink 14 to 15 was done 5-6 days ago.

Today, after adding a new host, my list of 24 hosts was emptied. It’s sad.

Actions I tryied without success after I lost my hosts list :

I slide off my iCloud sync. 
Close and restart Blink.
Activate iCloud sync.

Configuration :
IPad Pro
Ipados 15.3.1
Blink 15.0.5.526
ICloud sync ON (only this iPad with iCloud sync on)

Can I restore my host list and how?

In the futur, can we backup our hosts list?

Thank’s

**Comment by jaclu** [comment]
This just happened to me as well, On my iPad (Blink 15 latest) I added one host, and it turned out to be a duplicate name. I changed the new one to a unique name, once saved,  all my previous hosts  were gone.
I rebooted the iPad turned of and on iCloud sync, but the host list was stuck at empty. It did not seem to be a iCloud issue, my iPhone still showed all my hosts, a restart of the iPhone didn't change this. In the end I did a config sync from Blink 14 to get my iPad hosts kind of back, but then had to change stuff manually, and after I added the new host with a unique name, it did not sync over to my iPhone via iCloud.

How do I get both of my devices back to syncing settings?

**Comment by carloscabanero** [comment]
This was an issue when performing the migration prior to 526. When migrating, the iCloud IDs for the hosts were not reset, and as they were not matching they would be deleted. Doing the migration again with 526 should import everything and synchronize with iCloud without issue.

If you want to keep a backup of your hosts, just make a copy of the .blink/hosts file.

@jaclu Your issue is different though, I replied on that thread.

Closing here as fixed on #1386
